configs/projects/pe-bolt-server-runtime-main.rb

```diff
@@ -1,6 +1,6 @@
 project 'pe-bolt-server-runtime-main' do |proj|
   proj.setting(:pe_version, 'main')
-  proj.setting(:rubygem_puppet_version, '7.12.0')
+  proj.setting(:rubygem_puppet_version, '7.13.1')
```

Comment:
This also needs to get updated in configs/projects/pe-bolt-server-runtime-2019.8.x.rb and the bolt-runtime project file